File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/teba-tariff.ts
import { TebaTariffCategoryEnum } from "projects/shared-models-lib/src/lib/enums/teba-tariff-category-enum";

export class TebaTariff {
    tariffId: number;
    tariffCode: string;
    description: string;
    invoicingDescription: string;
    validFrom: string;
    validTo: string;
    costValue: number;
    minimumValue: number;
    adminFeePercentage: number;
    isDeleted: boolean;
    createdBy: string;
    createdDate: string;
    modifiedBy: string;
    modifiedDate: string;
    tebaTariffCategory: TebaTariffCategoryEnum | null;
    ruleDescription: string;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/TebaTariff.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class TebaTariff : Common.Entities.AuditDetails
    {
        public int TariffId { get; set; } // TariffId (Primary key)
        public string TariffCode { get; set; } // TariffCode (length: 50)
        public string Description { get; set; } // Description (length: 200)
        public string InvoicingDescription { get; set; } // InvoicingDescription (length: 150)
        public System.DateTime ValidFrom { get; set; } // ValidFrom
        public System.DateTime ValidTo { get; set; } // ValidTo
        public decimal CostValue { get; set; } // CostValue
        public decimal MinimumValue { get; set; } // MinimumValue
        public decimal AdminFeePercentage { get; set; } // AdminFeePercentage
        public TebaTariffCategoryEnum? TebaTariffCategory { get; set; } // TariffCategoryId
        public string RuleDescription { get; set; } // RuleDescription (length: 150)
    }
}
